Example/HelloWorld/Package.swift

```diff
@@ -0,0 +1,43 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "HelloWorld",
+    platforms: [
+        .macOS(.v13),
+        .iOS(.v16),
```

Comment:
Should probably be just iOS 17 in this case to match the deployment target in BUILD I believe